TrainingExtensions/torch/src/python/aimet_torch/onnx_utils.py

```diff
@@ -297,9 +297,10 @@ def symbolic(g, inp, identifier, start, is_leaf):
         """
         Magic method that helps with exporting a custom ONNX node
         """
+        output = g.op('aimet_torch::CustomMarker', inp, id_s=identifier, leaf_s=is_leaf, start_s=start)
+        output.setType(inp.type())
         # Note: the attribute are listed alphabetically in onnx.NodeProto
-        return g.op('aimet_torch::CustomMarker', inp, id_s=identifier, leaf_s=is_leaf, start_s=start)\
-                .setType(inp.type())
+        return output
```

Comment:
Hi @hnsyprst thanks for the PR.
However, what is the main change in this PR?
It doesn't seem to me that the behavior of this function will change before/after this PR, would it?